demo/app/lib/views/credential_preview.dart

```diff
@@ -47,7 +47,7 @@ class CredentialPreviewState extends State<CredentialPreview> {
         shrinkWrap: true,
         itemBuilder: (context, position) {
           //TODO Ignoring the photo value for now due to extremely long text need to render in a separate issue-881
```

Comment:
(not directly related to this, but where does this issue# come from? I can't find it)